aws/terraform/infra/data.tf

```diff
@@ -0,0 +1,6 @@
+/*
+ Grant us ability to yield different availability zones for a region
+*/
+data "aws_availability_zones" "available" {
+  state = "available"
+}
```

Comment:
Little micro-bug here.  For accounts using AWS Localzones, this occasionally returns those AZ's for the VPC deployment, and LZ doesn't support RDS or NLB at the moment.  A filter stanza like the following can correct that:
```
filter {
  name  = "opt-in-status"
  values = ["opt-in-not-required"]
}
```